packages/rendermime/src/latex.ts

```diff
@@ -186,6 +189,7 @@ function init() {
       processEscapes: true,
       processEnvironments: true
     },
+    TeX: { equationNumbers: { autoNumber: 'AMS' } },
```

Comment:
This PR is still an improvement even if you do not enable equation numbering.

That is true even if you don't deal with the global mathjax object problem.